networkx/algorithms/dominance.py

```diff
@@ -135,3 +135,37 @@ def dominance_frontiers(G, start):
                         df[v].add(u)
                         v = idom[v]
     return df
+
+
+def dominator_tree(G, start):
+    """Returns the dominator tree from a directed graph.
+
+
+    Parameters
+    ----------
+    G : a DiGraph or MultiDiGraph
+        The graph where dominance is to be computed.
+
+    start : node
+        The start node of dominance computation.
+
+    Returns
+    -------
+    T : DiGraph
+        Dominator tree where each node's children are those it immediately dominates
+
+    Raises
+    ------
+    NetworkXNotImplemented
+        If `G` is undirected.
+
+    NetworkXError
+        If `start` is not in `G`.
+
+    Notes
+    -----
+    Note that if a CFG has a single exit node,
+    then postdominance is equivalent to dominance if
+    flow is reversed (going from the exit node to the start node)
+    """
+    return nx.DiGraph(((v, u) for u, v in nx.immediate_dominators(G, start).items() if u != start))
```

Comment:
This line is too long (longer than 79 characters). There is also a redundant pair of parentheses. You can split it up into two lines as follows:

```python
    dominators = nx.immediate_dominators(G, start)
    return nx.DiGraph((v, u) for u, v in dominators.items() if u != start)
```